packages/x-date-pickers/src/internals/models/fields.ts

```diff
@@ -2,20 +2,21 @@ import * as React from 'react';
 import type { UseFieldInternalProps } from '../hooks/useField';
 import { FieldSection, PickerValidDate } from '../../models';
 import type { ExportedUseClearableFieldProps } from '../../hooks/useClearableField';
+import { MakeRequired } from './helpers';
 
 export interface BaseFieldProps<
   TValue,
   TDate extends PickerValidDate,
   TSection extends FieldSection,
   TEnableAccessibleFieldDOMStructure extends boolean,
   TError,
-> extends Omit<
+> extends MakeRequired<
```

Comment:
The format is always defined when a field is used inside a picker (I'm currently making sure it is, because it makes the customization easier) and so is the value